apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -283,13 +283,13 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         }
 
         if parse_error is None and name in data["submitted_answers"]:
-            a_sub = data["submitted_answers"].get(name)
+            submitted_answer = data["submitted_answers"].get(name)
             # If answer is in a format generated by pl.to_json, convert it
             # back to a standard type (otherwise, do nothing)
-            a_sub = pl.from_json(a_sub)
+            submitted_answer = pl.from_json(submitted_answer)
 
             html_params["suffix"] = suffix
-            html_params["a_sub"] = ("{:" + custom_format + "}").format(a_sub)
+            html_params["a_sub"] = ("{:" + custom_format + "}").format(submitted_answer)
```

Comment:
Change `a_sub` to `submitted_answer`?

Reply:
Just fixed [0b8c23c](https://github.com/PrairieLearn/PrairieLearn/pull/8846/commits/0b8c23c2246249ff9acf522651675ea566c82583), my bad; missed that one.